### ISSUE TITLE
Plugin broken

### ISSUE THREAD

**Issue by edge0703** [tjerabek/sketch-uifaces#2]
Your plugin seems to be broken in the latest version of Sketch (41.2):
I select a layer, go to "Plugin > UI Faces > Load random faces", but nothing happens. Even if I don't select a layer. I tried to un- and re-install the plugin, restart Sketch, nothing.

Here's a short video that shows, that nothing happens at all:
https://db.tt/9tMt7nLKOh

I get no (error) message at all. Also tried it in the beta version, nothing.

**Comment by ghost** [comment]
+1

**Comment by tjerabek** [comment]
It looks like UI faces API is broken. I'll try to contact the owner.

**Comment by matchboxhero** [comment]
Did you have any luck?

**Comment by edge0703** [comment]
+1

**Comment by tjerabek** [comment]
@matchboxhero No, I didn't. He didn't respond me. It seams I need to create a library without connection to the api.

**Comment by matchboxhero** [comment]
Thanks for the response! That's a shame.

**Comment by matchboxhero** [comment]
You've probably already seen it but this is on the site's about page;

> Update, October 2016
> Hi friends, I recieved a note from Amazon this week "EC2 has detected degradation of the underlying hardware hosting your Amazon EC2 instance [uifaces]. Due to this degradation, your instance could already be unreachable." I've set up this temporary version of UIFaces while I rebuild!
> 

That's probably the culprit.

**Comment by tjerabek** [comment]
I've seen it. I'm not sure why there is the temporary version.

**Comment by perfectflow** [comment]
I also needed this plugin badly, so created my own version of UI Faces and updated Tomáš's Sketch plugin. Check the new website at http://uifaces.co and get the plugin here https://github.com/perfectflow/sketch-uifaces

Maybe Tomáš will update his version too.

**Comment by tjerabek** [comment]
@perfectflow That's great! Is there any way how to redirect people to your project?

**Comment by perfectflow** [comment]
Hey Tomáš, nice to meet you. I'm not really familiar how GitHub works. I forked your project, do you maybe merge, or just add a link in readme that redirects to my version of the plugin?

I've just updated your plugin and added a few extra features as a start, but will work on few other improvements in the future.

Also the author in the manifest.json is still you, do I add myself as an author of my version or keep your details?
